src/components/Match/Match.js

```diff
@@ -78,20 +82,69 @@ const Match = ( {
             navToTaxonDetails={navToTaxonDetails}
           />
           { !suggestionsLoading
-    && (
-      <View className="mt-5">
-        <Heading3 className="mx-4">
-          {t( "Do-you-know-what-group-this-is-in" )}
-        </Heading3>
-        <IconicSuggestionsScroll
-          iconicTaxonChosen={iconicTaxon}
-          onIconicTaxonChosen={setIconicTaxon}
+          && (
+            <View className="mt-5">
+              <Heading3 className="mx-4">
+                {t( "Do-you-know-what-group-this-is-in" )}
+              </Heading3>
+              <IconicSuggestionsScroll
+                iconicTaxonChosen={iconicTaxon}
+                onIconicTaxonChosen={setIconicTaxon}
+              />
+              <Body2 className="mx-4 mt-7 mb-20">
+                {t( "If-you-took-the-original-photo-you-can-help" )}
+              </Body2>
+            </View>
+          )}
+        </ScrollViewWrapper>
+        <SaveDiscardButtons
+          handlePress={handleSaveOrDiscardPress}
         />
-        <Body2 className="mx-4 mt-7 mb-20">
-          {t( "If-you-took-the-original-photo-you-can-help" )}
-        </Body2>
-      </View>
-    )}
+      </>
+    );
+  }
+  // In case there are suggestions but no top suggestion
+  if ( !topSuggestion ) {
+    return (
+      <>
+        <ScrollViewWrapper scrollRef={scrollRef}>
+          <View className={cardClassTop}>
+            {
+              suggestionsLoading
+                ? (
+                  <ActivityIndicator size={33} />
+                )
+                : (
+                  <Body2>
```

Comment:
It looks like the styling of this is off just a touch from Figma. In Figma the padding above and below this text is 18px, and here it appears to be 16px.